x-pack/plugins/security_solution/common/detection_engine/rule_monitoring/model/log_level.ts

```diff
@@ -80,3 +81,17 @@ export const logLevelFromExecutionStatus = (status: RuleExecutionStatus): LogLev
       return LogLevel.trace;
   }
 };
+
+export const logLevelFromLastRunOutcome = (status: RuleLastRunOutcomes): LogLevel => {
```

Comment:
This looks to be unused. Were you going to use it later for something?

Reply:
I removed it as we don't need it right now